docs/examples.md

```diff
@@ -0,0 +1,35 @@
+# Examples
```

Comment:
This is the only top-level section where the sidebar only contains one item.

<img width="605" alt="Screenshot 2024-10-01 at 16 28 39" src="https://github.com/user-attachments/assets/7fd88325-498d-4b97-a431-daf3d4a48e75">

Reply:
```suggestion
---
hide:
- navigation
---
# Examples
```

Reply:
Thanks